packages/ssx-sdk/src/modules/Storage.ts

```diff
@@ -0,0 +1,42 @@
+import { IEncryption } from './Encryption';
+import { IUserAuthorization } from './UserAuthorization';
+
+interface IStorage {
```

Comment:
Does storage and datavault need to be seperate interfaces? Does this allow for some flexibility that other approaches don't?